xml/System/AppContext.xml

```diff
@@ -135,15 +138,15 @@
  To restore the method's previous behavior and prevent the exception, you can add the `Switch.System.IO.UseLegacyPathHandling` switch to the application configuration file for the example:  
   
 ```xml  
-  
 <configuration>  
     <runtime>  
         <AppContextSwitchOverrides value="Switch.System.IO.UseLegacyPathHandling=true" />    
     </runtime>  
 </configuration>  
-  
 ```  
-  
+## See also
```

Comment:
Off topic: I find it weird that See also links are showing up before the members table. It seems that none of our previous see also links were exported since it's the first time I'm seeing this behavior:
![image](https://user-images.githubusercontent.com/12971179/30833966-f2dff5f6-a205-11e7-8c2f-c7069a07c842.png)

Should we file a feature request to treat that similar to the examples section where it can move to a different location on the page and have it linked from the right-hand menu?

Reply:
@mairaw, yes, we should. See Also in that position strikes me as something that almost everyone will overlook.

Reply:
@rpetrusha I'll file a feature request. None of our See also got exported. Just confirmed.